src-web/components/kappnav/JobView.jsx

```diff
@@ -75,27 +75,35 @@ class JobView extends Component {
       return (
         <div>
           <SecondaryHeader title={viewTitle} location={location}/>
-          <div className="page-content-container" role="main">
 
-        
-        <ResourceTable
-          rows={this.state.rows}
-          headers={this.state.headers} title={''}
-          onInputChange={(e) => {
-            this.searchInputChange(e)
-          }}
-          totalNumberOfRows={this.state.filteredRows.length}
-          changeTablePage={(e) => {
-            this.handlePaginationClick(e)
-          }}
-          sortColumn={this.state.sortColumn}
-          sortDirection={this.state.sortDirection}
-          handleSort={(e) => {
-            this.handleSort(e)
-          }}
-          pageNumber={this.state.pageNumber}
-        />
-        </div>
+          <div className="page-content-container" role="main">
+            <ResourceTable
+              rows={this.state.rows}
+              headers={this.state.headers} title={''}
+              onInputChange={(e) => {
+                this.searchInputChange(e)
+              }}
+              totalNumberOfRows={this.state.filteredRows.length}
+              changeTablePage={(e) => {
+                this.handlePaginationClick(e)
+              }}
+              sortColumn={this.state.sortColumn}
+              sortDirection={this.state.sortDirection}
+              handleSort={(e) => {
+                this.handleSort(e)
+              }}
+              pageNumber={this.state.pageNumber}
+              createNewModal={() => {
+                return (
+                  <div>
+                    <Button small iconDescription={msgs.get('run.audit')} onClick={() => openActionModal(document.documentElement.getAttribute('appnavConfigmapNamespace'), 'kappnav', 'app-nav-inventory', "Run Inventory Command Action")}>
```

Comment:
I think you need to translate "Run Inventory Command Action'

Reply:
I'll add the message so it can be translated.